Fedora/input/xccdf/system/software/gnome.xml

```diff
@@ -0,0 +1,853 @@
+<Group id="gnome">
+<title>GNOME Desktop Environment</title>
+<description>
+GNOME is a graphical desktop environment bundled with many Linux distributions that
+allow users to easily interact with the operating system graphically rather than
+textually. The GNOME Graphical Display Manager (GDM) provides login, logout, and user
+switching contexts as well as display server management.
+<br/><br/>
+GNOME is developed by the GNOME Project and is considered the default
+Fedora Graphical environment.
+<br/><br/>
+For more information on GNOME and the GNOME Project, see <b>https://www.gnome.org</b>
+</description>
+
+<Rule id="enable_dconf_user_profile" severity="high">
+<title>Configure GNOME3 DConf User Profile</title>
+<description>
+By default, DConf provides a standard user profile. This profile contains a list
+of DConf configuration databases. The user profile and database always take the
+highest priority. As such the DConf User profile should always exist and be
+configured correctly. 
+<br /><br />
+To make sure that the user profile is configured correctly, the <tt>/etc/dconf/profile/user</tt> should be set as follows:
+<pre>user-db:user
+system-db:local
+system-db:site
+system-db:distro
+</pre>
+</description>
+<ocil clause="DConf User profile does not exist or is not configured correctly">
+To verify that the DConf User profile is configured correctly, run the following
+command:
+<pre>$ cat /etc/dconf/profile/user</pre>
+The output should show the following:
+<pre>user-db:user
+system-db:local
+system-db:site
+system-db:distro</pre>
+</ocil>
+<rationale>
+Failure to have a functional DConf profile prevents GNOME3 configuration settings
+from being enforced for all users and allows various security risks.
+</rationale>
+<oval id="enable_dconf_user_profile" />
+</Rule>
+
+<Group id="gnome_login_screen">
+<title>Configure GNOME Login Screen</title>
+<description>In the default GNOME3 desktop, the login is displayed after system boot
+and can display user accounts, allow users to reboot the system, and allow users to
+login automatically and/or with a guest account. The login screen should be configured
+to prevent such behavior.
+<br /><br />
+For more information about enforcing preferences in the GNOME3 environment using the DConf
+configuration system, see <b>https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Desktop_Migration_and_Administration_Guide/index.html</b> and the man page <tt>dconf(1)</tt>.
```

Comment:
Would it be possible to supply some Fedora equivalent URL to the above RHEL-7 one?